app/ui-react/packages/atlasmap-assembly/src/environments/environment.ts

```diff
@@ -3,6 +3,7 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
+  classpath: ' ',
```

Comment:
Is the space significant at all?  If so could you maybe pop some comments in here as to why?

Reply:
mostly because as @riccardo-forina has figured out, I'm extremely forgetful 😄

Reply:
ah good point, I'll add it - this is because AtlasMap triggers if (!classpath), and even empty classpath without space enter into that if clause. Yep, kinda dirty hack. I think Syndesis will never use maven classpath resolution though. It's incredibly slow.

Reply:
OK done